yarn.lock

```diff
@@ -1869,7 +1887,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@ledgerhq/hw-transport@npm:^5.17.0, @ledgerhq/hw-transport@npm:^5.36.0":
+"@ledgerhq/hw-transport@npm:^5.36.0":
```

Comment:
Can we just run a `yarn dedupe` (we have 2 versions of this package)

Reply:
Sorry for that. Fixed.